frontend/.storybook/preview.js

```diff
@@ -1,7 +1,7 @@
 import { theme as V5Theme } from '@/v5/ui/themes/theme';
```

Comment:
#### Updating a view with transformation by removing the transformation doesn't do anything

##### Steps
- Create a ticket with a view that contains a transform 
- Save the ticket
- Open the ticket, reset the transform, and update the view

##### current
- The view resets to what it was before saved (goes away after refresh, the data is sound just some unchanged state)?

##### expected
view should not move when I update the view

![24-01-19 01-30-47](https://github.com/3drepo/3drepo.io/assets/11945337/81ff450d-c0d0-4936-9210-fd1e97d6c1d5)